packages/common/src/apiutil/LionwebResponse.ts

```diff
@@ -30,7 +30,9 @@ export type ResponseMessage = {
  * @param object
  */
 export function isResponseMessage(object: unknown): object is ResponseMessage {
-    return object["kind"] !== undefined && typeof object["kind"] === "string"
+    // @ts-expect-error TS7053
+    return object["kind"] !== undefined && typeof object["kind"] === "string" &&
```

Comment:
would casting to `unknown` help avoiding the TS error?

Reply:
Not really, as you cannot access properties on an unknown object.  And object itself is already unknown.
But you gave me an idea, I can cast the object to ResponseMessage and then access its properties,